plugins/rabbitmq_logs.yaml

```diff
@@ -0,0 +1,34 @@
+version: 0.0.1
+title: RabbitMQ
+description: Log parser for RabbitMQ
+parameters:
+  - name: daemon_log_path
+    type: string
+    default: "/var/log/rabbitmq/rabbit@*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
```

Comment:
I think it might be worth exposing or setting the following:
- `include_file_path`
- `resource`

Im sure there are not semantic conventions for Rabbitmq logging, but if there were, I think there would be some default `resource` values.

Reply:
Those are good upgrades this initial push is just to get them ported over into the repo. We should apply best practices after this initial push. I think you probably know more about these than I do so I think we should sync up at some point and get these best practices applied.